### ISSUE TITLE
"refresh-outputs" is not currently supported by Compose.

### ISSUE THREAD

**Issue by zocom-johan-kivi** [serverless/serverless#12791]
### Issue description

When trying to refresh outputs in a composed services I get the following error. 

`The command "refresh-outputs" is not currently supported by Compose.`

I now use Serverless 4.2.4.
Worked fine in 3.8.x.

### Context

# serverless-compose.yml
```
services:
  base:
    path: base-services
  auth:
    path: auth-api
    dependsOn: base
    params:
      userDetailsTable: ${base.userDetailsTable}
      userDetailsTableName: ${base.userDetailsTableName}
      userPool: ${base.userPool}
      userPoolClient: ${base.userPoolClient}
  externalservices:
    path: services-api
    dependsOn: base
    params:
      contentTable: ${base.contentTable}
      contentTableName: ${base.contentTableName}
      userDetailsTable: ${base.userDetailsTable}
      userDetailsTableName: ${base.userDetailsTableName}
      learningRecordsTable: ${base.learningRecordsTable}
      learningRecordsTableName: ${base.learningRecordsTableName}
      userPool: ${base.userPool}
      userPoolClient: ${base.userPoolClient}
```
# serverless.yml
```
service: services-api
frameworkVersion: "4"
org: my-org

provider:
  name: aws
  runtime: nodejs20.x
  architecture: arm64
  region: eu-north-1
  httpApi:
    cors: true
    authorizers:
      serviceAuthorizer:
        type: request
        functionName: authorizer
        payloadVersion: "2.0"
        enableSimpleResponses: true

plugins:
  - serverless-iam-roles-per-function

build:
  esbuild:
    bundle: true
    minify: true
    exclude:
      - "@aws-sdk/*"

package:
  individually: true

functions:
  - ${file(./yaml/auth-functions.yml)}
  - ${file(./yaml/calendar-functions.yml)}
  ```

**Comment by skierkowski** [comment]
@zocom-johan-kivi sorry about that issue. I got some good news for you - we have a release coming shortly (this week or next week) with some big improvements to Serverless Compose. We are making remote state available so you can easily collaborate with your teammates, and you won't need to use `refresh-outputs` anymore.

I'm going to keep this issue open until those improvements are released so we can track this.

**Comment by skierkowski** [comment]
@zocom-johan-kivi we are about to release the new version with the shared state (in the next hour). The PR with the docs is available [here](https://github.com/serverless/serverless/pull/12780/files). Instead of saving the state in a local file it'll be saved in S3 by default, and therefore you will no longer need to use `refresh-outputs` as the outputs will be persisted in the S3 bucket.

> Will remote state impact or deployment pipeline you think?

I'm assuming you don't use `refresh-outputs` in the pipeline, but if you do, then this will unfortunately break the pipeline and you'll have to remove that command. Otherwise the rest of the process should be automatic. There are some edge cases, but those are addressed in the docs.

The release should be coming shortly, so I'm looking forward to your feedback.

**Comment by skierkowski** [comment]
@zocom-johan-kivi I'm going to close this issue out because we released 4.3.2 which includes support for the remote state & deployment bucket storage. Getting the new remote state should have been seamless, [here are the docs](https://www.serverless.com/framework/docs/guides/state) with details on how it works.

I'm going to close this issue out since this is now supported. However, please add additional comments with feedback.
